data/mpotane.json

```diff
@@ -18,7 +18,7 @@
     },
```

Comment:
I think you need to delete these two brackets & make it clear. It will help to read.

Reply:
Which one bro? I don't see brackets in the suggested changes to be made.

Reply:
![Screenshot from 2023-02-03 22-37-57](https://user-images.githubusercontent.com/88102392/216664227-45291db4-ec38-4fba-9996-275d695c31be.png)

Reply:
I think it doesn't show the lines 14 to 17. but it has properties in it.

Reply:
ok